### PR TITLE
Upgrade to Pulumi v1.0.0

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: resources.go [hunk 1]
```diff
@@ -146,6 +146,7 @@ func Provider() tfbridge.ProviderInfo {
 			"cloudflare_zone_lockdown":          {Tok: makeResource(mainMod, "ZoneLockdown")},
 			"cloudflare_zone_settings_override": {Tok: makeResource(mainMod, "ZoneSettingsOverride")},
 			"cloudflare_zone":                   {Tok: makeResource(mainMod, "Zone")},
+			"cloudflare_custom_ssl":             {Tok: makeResource(mainMod, "CustomSsl")},
```
Comment (pgavlin) [inline resources.go]: Is this a bugfix?

Comment (stack72) [reply resources.go]: yes - it was missing from the SDK generation so I added it now

Comment (pgavlin) [reply resources.go]: Okay. Might add that to the release notes as well.

Comment (stack72) [reply resources.go]: +1 will add when releasing to stop another set of builds

**Pull request merged**